launcher/CMakeLists.txt

```diff
@@ -83,18 +90,26 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 if(TARGET Qt6::Core)
 	qt_wrap_ui(launcher_UI_HEADERS ${launcher_FORMS})
+	if(ENABLE_TRANSLATIONS)
+		qt_add_translation( launcher_QM ${launcher_TS} )
```

Comment:
consider using `qt_add_translations`. From [qt_add_translation](https://doc.qt.io/qt-6/qtlinguist-cmake-qt-add-translation.html):

> Warning: This function is deprecated. Consider using the target-based functions [qt6_add_lrelease](https://doc.qt.io/qt-6/qtlinguist-cmake-qt-add-lrelease.html#qt6-add-lrelease) or [qt6_add_translations](https://doc.qt.io/qt-6/qtlinguist-cmake-qt-add-translations.html#qt6-add-translations) instead.

Reply:
Added TODO in code.
That function looks to be quite different from the old one. Don't feel confident enough to correctly change code that is not tested by either CI or by me.

Reply:
agree